plugins/net_plugin/net_plugin.cpp

```diff
@@ -759,29 +756,22 @@ namespace eosio {
       explicit msg_handler( const connection_ptr& conn) : c(conn) {}
 
       void operator()( const signed_block& msg ) const {
-         EOS_ASSERT( false, plugin_config_exception, "operator()(signed_block&&) should be called" );
+         EOS_ASSERT( false, plugin_config_exception, "Not implemented, call handle_message directly instead" );
```

Comment:
can this not be a templated `operator()` that throws the assert and catches all overloads that are not explicitly specified?

Reply:
Yes, it can now that no value category of type `packed_transaction[_ptr]` and `signed_block[_ptr]` are allowed.